connectors/src/connectors/intercom/temporal/client.ts

```diff
@@ -0,0 +1,65 @@
+import type { ModelId } from "@dust-tt/types";
+import type { WorkflowHandle } from "@temporalio/client";
+import { WorkflowNotFoundError } from "@temporalio/client";
+
+import { QUEUE_NAME } from "@connectors/connectors/intercom/temporal/config";
+import { intercomHelpCentersSyncWorkflow } from "@connectors/connectors/intercom/temporal/workflows";
+import { dataSourceConfigFromConnector } from "@connectors/lib/api/data_source_config";
+import { Connector } from "@connectors/lib/models";
+import { getTemporalClient } from "@connectors/lib/temporal";
+import logger from "@connectors/logger/logger";
+
+function getIntercomHelpCentersSyncWorkflowId(connectorId: ModelId) {
+  return `intercom-sync-help-centers-${connectorId}`;
+}
+
+export async function launchIntercomHelpCentersSyncWorkflow({
+  connectorId,
+}: {
+  connectorId: ModelId;
+}) {
+  const client = await getTemporalClient();
+  const connector = await Connector.findByPk(connectorId);
+  if (!connector) {
+    throw new Error(
+      `[Intercom] Connector not found. ConnectorId: ${connectorId}`
+    );
+  }
+
+  const dataSourceConfig = dataSourceConfigFromConnector(connector);
+  const workflowId = getIntercomHelpCentersSyncWorkflowId(connectorId);
+
+  try {
+    const handle: WorkflowHandle<typeof intercomHelpCentersSyncWorkflow> =
+      client.workflow.getHandle(workflowId);
+    try {
+      await handle.terminate();
+    } catch (e) {
+      if (!(e instanceof WorkflowNotFoundError)) {
+        throw e;
+      }
+    }
+    await client.workflow.start(intercomHelpCentersSyncWorkflow, {
+      args: [{ connectorId, dataSourceConfig }],
+      taskQueue: QUEUE_NAME,
+      workflowId: workflowId,
+      searchAttributes: {
+        connectorId: [connectorId],
+      },
+      cronSchedule: "0 * * * *", // every hour
```

Comment:
I'm not sure what's the best delay, I don't expect Help Center to change super often so we could probably put more delay, maybe even once a day. WDYT?

Reply:
cc @fontanierh @lasryaric to comment on the approach based on previous experience.

So that means we'll run something every hour for all users. Will we see the workflow in the temeporal app when it's not running? How do we stop it?

Reply:
We already do that for the GoogleDriveWebhook renew workflow.
It's basically another way to start a workflow and have it Temporal.sleep() the right amount of time.
It's showing up on the Temporal UI like any workflow, it's sleeping, you can kill it etc.